scripts/wallet.py

```diff
@@ -9,9 +9,12 @@
 import logging
```

Comment:
I've heard about this file from Aaro so I won't comment :D

Reply:
XD